src/daria/image/image.py

```diff
@@ -353,14 +360,98 @@ def copy(self) -> "Image":
         """
         Returns a copy of the image object.
         """
-        return Image(self.img, self.origo, self.width, self.height)
+        return Image(self.img, self.metadata, color_space=self.colorspace)
 
     def toBGR(self) -> None:
-        if self.colorspace == "rgb":
+        """
+        Transforms image to BGR if it is in RGB
+        """
+        if self.colorspace == "RGB":
             self.img = cv2.cvtColor(self.img, cv2.COLOR_BGR2RGB)
-            self.colorspace = "bgr"
+            self.colorspace = "BGR"
+            self.metadata["color_space"] = "BGR"
 
     def toRGB(self) -> None:
-        if self.colorspace == "bgr":
+        """
+        Transforms image to RGB if it is in BGR
+        """
+        if self.colorspace == "BGR":
             self.img = cv2.cvtColor(self.img, cv2.COLOR_BGR2RGB)
-            self.colorspace = "rgb"
+            self.colorspace = "RGB"
+            self.metadata["color_space"] = "RGB"
+
+    def toGray(self) -> da.Image:
+        """
+        Returns a greyscale version of the daria image
+        """
+        gray_img = self.copy()
+        if self.colorspace == "RGB":
+            gray_img.img = cv2.cvtColor(gray_img.img, cv2.COLOR_RGB2GRAY)
+        elif self.colorspace == "BGR":
+            gray_img.img = cv2.cvtColor(gray_img.img, cv2.COLOR_BGR2GRAY)
+        elif self.colorspace == "GRAY":
+            pass
+        else:
+            raise Exception(
+                "Only RGB or BGR images can be converted to Gray at the moment"
+            )
+        gray_img.colorspace = "GRAY"
+        gray_img.update_metadata()
+        return gray_img
+
+    def toRed(self) -> da.Image:
+        """
+        Returns a red channel version of the daria image
+        """
+        red_img = self.copy()
+        if self.colorspace == "RGB":
+            red_img.img = red_img.img[:, :, 0]
+        elif self.colorspace == "BGR":
+            red_img.img = red_img.img[:, :, 2]
+        elif self.colorspace == "RED":
+            pass
+        else:
+            raise Exception(
+                "Only RGB or BGR images can be converted to Red at the moment"
+            )
+        red_img.colorspace = "RED"
```

Comment:
Wouldn't `R` instead of `RED` be more consistent to `RGB`? I am OK (and maybe even in favor) with using `RED` as it is simpler to search for it in the code, but thought to raise this question for possible rethinking nevertheless. Before, we switch too often conventions.

Reply:
Let us discuss this briefly as well, I do not have a too strong opinion, but it felt to me like "RGB" means something, while "R" alone does not. Therefore, I chose "RED". I am not sure if capital letters are the way to go in either of the places here though. What do you think?

Reply:
Let's keep `RED`. Indeed, it is undoubted interpretable without further explanation, which is the most important. Please resolve.